components/library/libArcus/Makefile

```diff
@@ -40,5 +41,5 @@ install: $(INSTALL_32_and_64)
 
 test: $(NO_TESTS)
 
-REQUIRED_PACKAGES += runtime/python-34
 REQUIRED_PACKAGES += library/c++/protobuf
+REQUIRED_PACKAGES += $(PYTHON3_RUNTIME_PKG)
```

Comment:
Is history file intentionally left blank?

Reply:
There is no deprecation here, just that the default python runtime has changed since 3.5 was made default and the detected fmri in REQUIRED_PACKAGES is runtime/python-35.
Since the package was not rebuilt it did not show.

Reply:
@pyhalov Do you think we can merge this one?

Reply:
Likely. Why don't remove python 3.4 itself?

Reply:
Python 3.4 removal is in another PR to merge after this one.
I wanted to do it in two stages to avoid making mistakes.